#! /bin/sh /usr/share/dpatch/dpatch-run
## 20_dont_overwrite_pdl_1p.dpatch by Henning Glawe <glaweh@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: work around pdl.1p (generated from Basic/Pod/Intro.pod) being 
## DP: overwritten by empty text autogenerated from pdl.PL

@DPATCH@
diff -urNad pdl-2.4.2/Makefile.PL /tmp/dpep.0RhMw8/pdl-2.4.2/Makefile.PL
--- pdl-2.4.2/Makefile.PL	2005-01-15 13:36:35.000000000 +0100
+++ /tmp/dpep.0RhMw8/pdl-2.4.2/Makefile.PL	2005-01-15 13:37:55.000000000 +0100
@@ -391,6 +391,8 @@
 	      'VERSION_FROM' => 'Basic/Core/Version.pm',
 	      'EXE_FILES' => ['pdl', 'perldl', 'pdldoc'],
 	      'PM' => { @podpms }, # so that the script docs are picked up
+              'MAN1PODS' => { 'perldl' => '$(INST_MAN1DIR)/perldl.$(MAN1EXT)', 
+                              'pdldoc' => '$(INST_MAN1DIR)/pdldoc.$(MAN1EXT)'},
 	      'MAN3PODS' => {}, # don't pick up the script pods again
 	      'OPTIMIZE'  => $PDL::Config{OPTIMIZE} || $Config{optimize},
 	      'linkext'  => {LINKTYPE => ''},  # No linking required
